test/integration/host_test.rb

```diff
@@ -24,15 +24,6 @@ class HostIntegrationTest < ActionDispatch::IntegrationTest
     end
   end
 
-  test "destroy redirects to hosts index" do
```

Comment:
there's a similar test in [host_js_test](https://github.com/theforeman/foreman/pull/9002/files#diff-a329d2e69a2d86023bc418d5634d1f39fa2a73c9e4bdc0da8efb0af83dd86a4cR102)